exadata-database-service-on-cc/introduction/introduction.md

```diff
@@ -0,0 +1,81 @@
+# Introduction
+
```

Comment:
@leoneila Add ## About this Workshop heading

Reply:
added changes for the ## About this workshop